Formula/f/foldseek.rb

```diff
@@ -0,0 +1,50 @@
+class Foldseek < Formula
+  desc "Search your protein structures in seconds"
+  homepage "https://github.com/steineggerlab/foldseek"
+  url "https://github.com/steineggerlab/foldseek/archive/8-ef4e960.tar.gz"
+  version "8-ef4e960"
+  sha256 "c74d02c4924d20275cc567783b56fff10e76ed67f3d642f53c283f67c4180a1e"
+  license "GPL-3.0-or-later"
+  head "https://github.com/steineggerlab/foldseek.git", branch: "master"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "rust" => :build
+
+  uses_from_macos "bzip2"
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "libomp"
+  end
+
+  resource "testdata" do
```

Comment:
```suggestion
  resource "homebrew-testdata" do
```

Reply:
Thank you for your review. I'll modify it.